scripts/setup.sh

```diff
@@ -29,8 +29,15 @@ $DOCKER_COMPOSE exec rabbitmq rabbitmqctl set_permissions --vhost omotes root ".
 $DOCKER_COMPOSE exec rabbitmq rabbitmqctl add_vhost omotes_celery
 $DOCKER_COMPOSE exec rabbitmq rabbitmqctl set_permissions --vhost omotes_celery root ".*" ".*" ".*"
 
-$DOCKER_COMPOSE exec rabbitmq rabbitmqctl add_user --vhost omotes ${RABBITMQ_OMOTES_USER_NAME} ${RABBITMQ_OMOTES_USER_PASSWORD}
-$DOCKER_COMPOSE exec rabbitmq rabbitmqctl set_permissions --vhost omotes ${RABBITMQ_OMOTES_USER_NAME} ".*" ".*" ".*"
-$DOCKER_COMPOSE exec rabbitmq rabbitmqctl add_user --vhost omotes_celery ${RABBITMQ_CELERY_USER_NAME} ${RABBITMQ_CELERY_USER_PASSWORD}
-$DOCKER_COMPOSE exec rabbitmq rabbitmqctl set_permissions --vhost omotes_celery ${RABBITMQ_CELERY_USER_NAME} ".*" ".*" ".*"
+$DOCKER_COMPOSE exec rabbitmq rabbitmqctl add_user --vhost omotes "${RABBITMQ_OMOTES_USER_NAME}" "${RABBITMQ_OMOTES_USER_PASSWORD}"
+$DOCKER_COMPOSE exec rabbitmq rabbitmqctl set_permissions --vhost omotes "${RABBITMQ_OMOTES_USER_NAME}" ".*" ".*" ".*"
+$DOCKER_COMPOSE exec rabbitmq rabbitmqctl add_user --vhost omotes_celery "${RABBITMQ_CELERY_USER_NAME}" "${RABBITMQ_CELERY_USER_PASSWORD}"
+$DOCKER_COMPOSE exec rabbitmq rabbitmqctl set_permissions --vhost omotes_celery "${RABBITMQ_CELERY_USER_NAME}" ".*" ".*" ".*"
 
+# Add influxdb users with write access for optimizer/simulator and with admin rights for the frontend (root admin user via env vars)
+$DOCKER_COMPOSE up -d --wait omotes_influxdb
+$DOCKER_COMPOSE exec omotes_influxdb influx -username "${INFLUXDB_ADMIN_USER}" -password "${INFLUXDB_ADMIN_PASSWORD}" -port "${INFLUXDB_PORT}" -execute "CREATE USER ${INFLUXDB_WRITE_USER} WITH PASSWORD '${INFLUXDB_WRITE_USER_PASSWORD}'"
```

Comment:
If the password for a user changes, it will not be changed by `CREATE USER`:
```bash
root@8c7b566192ca:/# influx -execute "CREATE USER todd WITH PASSWORD 'influxdb41yf3';"
root@8c7b566192ca:/# influx -execute "CREATE USER todd WITH PASSWORD 'influxdb41yf3';"
root@8c7b566192ca:/# influx -execute "CREATE USER todd WITH PASSWORD 'influxdb41yf4';"
ERR: user already exists
Warning: It is possible this error is due to not setting a database.
Please set a database with the command "use <database>".
user already exists
```

The error on 'user already exists' can just be skipped, no need to change the query for this case. However, we do need to change the passwords which may be done with `SET PASSWORD` (https://docs.influxdata.com/influxdb/v1/administration/authentication_and_authorization/#general-admin-and-non-admin-user-management). So my suggestion is to just add:
```bash
$DOCKER_COMPOSE exec omotes_influxdb influx -username "${INFLUXDB_ADMIN_USER}" -password "${INFLUXDB_ADMIN_PASSWORD}" -port "${INFLUXDB_PORT}" -execute "SET PASSWORD FOR '${INFLUXDB_WRITE_USER}' = '${INFLUXDB_WRITE_USER_PASSWORD}'"
```

Same with `INFLUXDB_FRONTEND_ADMIN_USER`